### DISCUSSION TITLE
I cannot create a Vercel account.

### DISCUSSION THREAD

**Discussion by mNdrew** [vercel/community#4115] [Account Issue] [answered]
### Question

I cannot create an account in Vercel because I always receive this error `The verification phone number has been temporarily blocked. Please wait 12 hours and try again.` when inputting my number for verification. I tried sending the team an email however I am yet to hear any response.

**Comment by amyegan** [comment]
Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. 

I hope you are no longer waiting for a response. But, if you still do not have a response, I would be happy to check on your ticket's status if you can share the case number here.

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Reply by a2zbits** [reply]
Same is happened to me just a while ago. I was trying to signup my account for the first time. I used GitHub to signup and I think my Vercel account was also created, but then it got stuck on the mobile verification screen. I am getting the same error. Please see the attached screen for reference. I think this is a general/common error and this should be given priority. In case it helps, I used a mobile number from Pakistan.

<img width="445" alt="image" src="https://github.com/vercel/community/assets/11661091/89a6f1d2-e243-4313-af12-0487223662af">

**Reply by mNdrew** [reply, marked as answer]
I already got my problem resolved. I emailed their registration support email (registration@vercel.com) regarding the issue that I am having and they got back to me in around 1-2 days with my account created. 

@a2zbits you may try contacting them through their email registration@vercel.com about the issue.

**Comment by caiyuan3859** [comment]
<img width="374" alt="Snipaste_2023-10-03_22-57-53" src="https://github.com/vercel/community/assets/146581397/1322eaab-cc0c-4b41-985d-af0b8658d696">

case number is 00163166

**Reply by amyegan** [reply]
The team is handling an unusually high number of cases right now. Because tickets are processed by the order in which they are received, it will take some time for the team to respond to all of the cases that were opened before yours 🙇
